src/cache.c

```diff
@@ -12,6 +12,13 @@ struct cache_entry *alloc_entry(char *path, char *content_type, void *content, i
     ///////////////////
     // IMPLEMENT ME! //
     ///////////////////
+    struct cache_entry *ce = malloc(sizeof(struct cache_entry));
```

Comment:
you can simply do malloc(sizeof(ce)) here since ce is already a cache_entry struct